packages/vue-virtual/src/index.ts

```diff
@@ -19,25 +27,25 @@ function useVirtualizerBase<TScrollElement, TItemElement extends Element>(
   const opts = unref(options)
   const virtualizer = new Virtualizer(opts)
   const state = shallowRef(virtualizer)
+  let doClean: (() => void) | undefined
 
-  const applyOptions = (() => {
-    let doClean: (() => void) | undefined
-    return (options: VirtualizerOptions<TScrollElement, TItemElement>) => {
-      doClean?.()
-      virtualizer.setOptions({
-        ...options,
-        onChange: (instance) => {
-          // Force an update event
-          triggerRef(state)
-          options.onChange?.(instance)
-        },
-      })
+  const applyOptions = (
+    options: VirtualizerOptions<TScrollElement, TItemElement>,
+  ) => {
+    doClean?.()
```

Comment:
I analysed other adapter and the core lib and I don't think we need to call doClean on each applyOptions call. It should only be called on unMounted for cleanup. What do you think?